### DISCUSSION TITLE
Nomens for work groups

### DISCUSSION THREAD

**Discussion by GordonDunsire** [crystalyragui/MARC2RDA#428] [General]
Most operational applications that use rdaw:P10399 "appellation of work group" and its element subtypes will use the nomen string as the value:

ex:work1 rdaw:P10399 "Work group title" . // appellation of work group
ex:work2 rdaw:P10399 "Work group title" .
ex:work3 rdaw:P10399 "Work group title" .

There may be some applications that use a nomen IRI, but this requires a distinct IRI for each occurrence because the entities are different:

ex:work1 rdaw:P10399 ex:nomen1 . // appellation of work group
ex:work2 rdaw:P10399 ex:nomen2 .
ex:work3 rdaw:P10399 ex:nomen3 .
ex:nomen1 rdan:P80068 "Work group title" . // nomen string
ex:nomen2 rdan:P80068 "Work group title" .
ex:nomen3 rdan:P80068 "Work group title" .

This is overkill unless the nomens require additional statements:

ex:nomen1 rdan:P80073 "The national library" . // assigned by agent
ex:nomen2 rdan:P80073 "The national library" .
ex:nomen3 rdan:P80073 "The national library" .

If the simpler nomen string approach is used, it cannot be maintained in the context of nomens. Instead, RDA expects the application to develop and maintain a value vocabulary (typically using SKOS) that is the source of the nomen strings:

ex:workGroup1 skos:prefLabel "Work group title"@en .
ex:workGroup1 skos:inScheme ex:workGroupVES .

Note that if the IRI of the vocabulary entry is used directly, there may be unintended consequences:

ex:work1 rdaw:P10399 ex:workGroup1 . // Implies ex:workGroup1 is an RDA Nomen and a SKOS Concept.

This requires further investigation in the context of [Questions for LC Net Dev](https://github.com/uwlib-cams/MARC2RDA/discussions/388) and specifically the modelling of name authority files.

**Comment by pan-zhuo** [comment]
Thank you @GordonDunsire for creating the discussion!

I'm thinking that "appellation of work group" is a way to collocate a set of similar (aggregating) works that have more or less the same set of aggregated expressions (all editions of "Complete works of Shakespeare"), or the sets of aggregated expressions may vary quite a bit but the publisher/aggregator(s) intend them to be "editions" of the "same" work ("Norton Anthology of English Literature"). There seems to be little guidance in RDA on how to determine work groups, so I would first like to confirm if my understanding is correct. I also wonder if RSC has plans to develop instructions on work groups, or if this will be something more community-driven and specific to particular vocabulary encoding schemes.

> There may be some applications that use a nomen IRI, but this requires a distinct IRI for each occurrence because the entities are different

You seem to suggest that collocation is achieved through string matching in most cases, and not through a common nomen entity because an "appellation relationship" is an entity boundary for nomens. This is along the lines of what I was thinking when I asked if this constraint could be loosened. It seems to me that using value vocabulary IRIs has a similar problem: it feels like repeating the same "nomen" IRI without explicitly saying it's a nomen.

<br>

On a different note, something that confuses me is the inconsistency in the relationship hierarchy of work group elements. ["Nomen: is authorized access point for work group of"](http://rdaregistry.info/Elements/n/P80173) is a subproperty of ["Nomen: is authorized access point for work of"](http://www.rdaregistry.info/Elements/n/P80079), which means an AAP for a work group is also an AAP for a work, and by definition this is true. However, the inverse isn't. ["Work: has authorized access point for work group"](http://rdaregistry.info/Elements/w/P10401) is not directly related to ["Work: has authorized access point for work"](http://rdaregistry.info/Elements/w/P10331). Ditto for identifiers. Am I missing something here?

**Reply by GordonDunsire** [reply]
@pan-zhuo: The work group approach is intended to be completely general. The main use cases include collocating works that are part of a "WE lock" (i.e. aggregating works) or the "WEM lock" (i.e. diachronic works); "super works" that stimulate multiple works (e.g. "Romeo and Juliet" the dramatic work, the moving picture works, the still image works, the musical works, etc.); works that have shared characteristics (this overlaps with "category of work"); and so on. That is why RDA is so vague.

The approach was introduced in 3R to meet this functionality. We considered adding a whole new entity Work Group or Super Work, but that is a very expensive solution for RDA and entity-based cataloguing. Furthermore, we knew that IFLA intended to remove such entities from the object-oriented version of LRM. Another method is to add a specific element for "has work group" to the Work entity, but the semantics aren't very distinct. So we settled on using the "appellation" relationship, primarily as nomen string, but allow it to be reified as Nomen for special applications.

It is easy to get distracted by Nomen; its entity-fication as a reification of the appellation statement obscures the functionality of the appellation statement, which is the use of a human-readable string label for a thing (instance). The reification is only useful in implementation scenario A (linked open data), and then only for "name control". I find it useful to think of Nomen as a "mapping" of a set of things (instances) and a set of strings (appellations of things). The set of things is described in RDF with the RDA element set; the set of strings is typically described in RDF with SKOS. Each thing2string combination is unique, and is reified as Nomen.

I agree that the relationship hierarchy is inconsistent:

ex:nomen1 rdano:P80173 ex:work1 . // is authorized access point for work group of
=> ex:work1 rdawo:P10401 ex:nomen1 . // has authorized access point for work group; entailed from inverse property relationship
=> ex:nomen1 rdano:P80079 ex:work1 . // is authorized access point for work; entailed from sub-property relationship
==> ex:work1 rdawo:P10331 ex:nomen1 . // has authorized access point for work; entailed from inverse propety relationship

=>rdawo:P10401 rdfs:subPropertyOf rdawo:P10331 . // entailed from inverse sub-property relationships; this is not declared in RDA Registry

I think it is true that "a nomen that is an access point of work that is selected for preference in a specific vocabulary encoding scheme for work groups" is also "a nomen that is an access point of work that is selected for preference in a specific vocabulary encoding scheme", so the last entailment is valid.

If we agree, I can propose to the RSC Technical Working Group that the missing subproperty relationships (for identifier and appellation as well as authorized access point) are added to the Registry for consistency. Or am I missing something?

**Reply by pan-zhuo** [reply]
> If we agree, I can propose to the RSC Technical Working Group that the missing subproperty relationships (for identifier and appellation as well as authorized access point) are added to the Registry for consistency. Or am I missing something?

Yes, thank you @GordonDunsire ! And also the corresponding changes to the Toolkit, if those aren't automated already.

**Reply by lake44me** [reply]
Sorry not to have been able to get to it this week, but I am confused.  Is a "Work group" a subclass of "Work"?  Or a superclass?   It doesn't seem to have been entified.  It feels to me that having the sub-property relations and their inverses between the two nomen types could confuse "Work" and "Work group" which, even though "Work group" is loosely defined, are two different things.

**Reply by CECSpecialistI** [reply]
I am also confused and looking forward to this discussion tomorrow. It seems that a "Work group" is simply a number of distinct Works which are closely related (derivative works based on the same work, diachronic works, aggregating works) and share a common appellation (a collective title in Original RDA speak?). It seems that without the ability to entify the group, each work gets assigned a duplicate appellation, which has the potential to create many entities for what is essentially the same Nomen (the equivalent of a collective title?). So, in this case, it might make the most sense to use literal values rather than minting Nomens. Why can't a Work with a whole-part relationship to the works in the Work Group be assigned this nomen, and have a Nomen entity minted for it? I feel like my understanding is incomplete.

**Comment by CECSpecialistI** [comment]
I am hopeful that someone else might be willing to sum up this discussion at the start of this agenda item tomorrow since I definitely won't know what I'm talking about. @GordonDunsire @pan-zhuo, any takers?

**Reply by GordonDunsire** [reply]
A work group is a set of works that an application wants to cluster for some purpose. The purpose is local and subjective, and is not an absolute characteristic of the works. Use cases include the "super-work" concept that is talked about in some communities; an example is a cluster of works based on the concept and story of "Romeo and Juliet", including Shakespeare's play, numerous film versions and other adaptations, and possibly the works that inspired Shakespeare in the first place. Another use case is the set of diachronic works that realize a serial through changes of title, splits, mergers, etc.

All of the use cases can be accommodated by a chain of works: Shakespeare's Romeo and Juliet is adapted as Bernstein's West Side Story is adapted as Spielberg's West Side Story is adapted as A.N.Other's graphic novel ...; Serial A is succeeded by Serial B is split into Serial C is merged to form Serial D ...

But RDA was asked to provide a cluster mechanism as a short-hand representation of the chain: (Serial A, Serial B, Serial C, ...) = work cluster 1. The term "cluster" was not used because it has a different meaning within OCLC's attempts to FRBRize WorldCat.

A work group is not a Work; a set or group or collection of persons is not a Person (it's a Collective Agent). There is no whole-part relationship in the LRM sense; a Person is not a part of a Collective Agent, which is why RDA uses "member" relationships. It is important to understand the LRM representation of whole-part relationships and the difference with aggregates and collections; it is the focus of the dissention in the report of the IFLA Working Group on Aggregates that contributed to the delay in full RDA implementation of FRBR/FRAD/FRSAD until the LRM consolidation.

RDA found no justification for adding a new, non-LRM entity for Work Group; the required functionality is accommodated by chains of existing entities and relationships.

Instead, RDA provides a set of relationships for assigning a common/collective appellation to the cluster. This is an extension of the LRM appellation relationship; the distinct relationships allow an application to keep the subjective appellations that are assigned by a cataloguing agency separate from, we hope ***, the objective appellations that are globally assigned during cataloguing.

In practice, it is the nomen string of the work group that is more useful for meeting the cluster requirement; as we have discussed, assigning a nomen requires a distinct nomen for each member of the cluster, which defeats the short-hand utility. By only using a nomen string, the cluster is formed by indexing the string; "Romeo and Juliet", filtered by the RDA work group appellation element, retrieves all works referenced by the string irrespective of the actual authorized access points for the works in the set. Similarly, the work group appellations can be filtered out of 'standard' authorized  access point browse lists, etc.

This is all irrelevant to the transform, because a work group is application-dependent.

*** Of course, LRM and RDA acknowledge that the boundary of a work varies from community to community ... so this is a somewhat vain 'hope' for top-down universal bibliographic control ...
